src/pages/LandingPage/index.tsx

```diff
@@ -22,7 +22,7 @@ export default function LandingPage() {
 		if (user) {
 			navigate("/dashboard");
 		}
-	});
+	}, [user, navigate]);
```

Comment:
I think we can disable the button on landing page whenever the user has clicked it. For some reason after I sign in with google, the button still clickable and then it brings me to the google sign in page again

Reply:
The button state dissapear when it redirects to the google sign in page